### ISSUE TITLE
Pattern: Link list visual design & design specs provided by Reboot designers

### ISSUE THREAD

**Issue by RobertaJHahn** [carbon-design-system/ibm-dotcom-library#2030]
#### Notes

  -  The Reboot team is responsible for creating the visual designs and design specs for most of the patterns included in the Solutions page.  This issue is to track the transition of the Reboot design work to the DDS design & engineering team members.

#### Additional Information
 - [Visual Designs & Design Specs Box link](https://ibm.ent.box.com/folder/101876058407)
- [Airtable](https://airtable.com/tblsnDDtb57UVxHpH/viwL7XPwcIgnb8Y4T?blocks=hide)
 - Link list [[storybook](http://ibmdotcom-react.mybluemix.net/?path=/story/patterns-sub-patterns-linklist--default)] [[code](https://github.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/react/src/patterns/sub-patterns/LinkList)] [[directory](https://ibm.ent.box.com/folder/101876058407)]
 - CTA [[storybook](http://ibmdotcom-react.mybluemix.net/?path=/story/components-cta--default)] [[code](https://github.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/react/src/components/CTA)] [[directory](https://ibm.ent.box.com/folder/101164203823)]

![image](https://user-images.githubusercontent.com/3793636/79607747-5fa25280-80b9-11ea-8eb0-3eaeda4df043.png)

Basically right now the link list only allows for a card style vertical list of links. We're thinking we can add multiple styles of links (See CTA component) like link with icon. This update will allow us to use this consistently accross some of the newer designs coming from the solution page work. Things like table of contents is an example that would leverage this update.

 - Horizontal layout
 - Style: text link
 - optional heading

> Need to finalize an optional eyebrow to the card style

#### Acceptance Criteria
 - [ ] Visual designs and specs received from Reboot designers
- [ ]  A pdf file is created and placed in the Box folder with the updated Sketch file
 - [ ] Meetings conducted with DDS team members to review the designs

**Comment by ltitle** [comment]
@RobertaJHahn Visual design specs (sketch and PDF) are now posted:
https://ibm.ent.box.com/folder/101876058407

**Comment by RobertaJHahn** [comment]
5/5/2020:  Visual designs reviewed with the DDS designers and engineers.
